app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+
+<div class="container">
+  <div class="row">
+
+    <% @hairstylists.each do |hairstylist| %>
+
+      <div class="col-xs-12 col-sm-4">
+      <h2><%= hairstylist.first_name %></h2>
+      <p><%= hairstylist.services %></p>
```

Comment:
I think it should be:
`hairstylist.services.name`

Reply:
No I think it should be
```ruby
hairstylist.services.each do |service|
...
end
```

Reply:
Actually:
`<ul>`
  `services.each do |service|`
    <li>`service.name`</li>
  `end`
`</ul>`